src/common/log.js

```diff
@@ -2,6 +2,7 @@
 
 const { join } = require('path')
 const { assign } = Object
+const { existsSync, mkdirSync } = require('fs')
```

Comment:
We will likely need to use `mkdirp` here then you don't need to use `exists` either, which is discouraged (deprecated even?) anyway. I.e., `const { sync: mdkir } = require('mkdirp')`

Since the logging runs at startup I'd prefer to use the async version, but since pretty much everything depends on the logger it's probably OK to make an exception.